packages/hash/frontend/src/components/layout/PageHeader/SearchBar/SearchInput.tsx

```diff
@@ -0,0 +1,144 @@
+import { Box, InputAdornment, InputBase, Tooltip } from "@mui/material";
+import { useRef } from "react";
+import { useKeys } from "rooks";
+import { faXmark } from "@fortawesome/free-solid-svg-icons";
+
+import { SearchIcon, FontAwesomeSvgIcon } from "../../../icons";
+
+export const SearchInput: React.FC<{
+  displayedQuery: string;
+  isMobile: boolean;
+  setResultListVisible: (visible: boolean) => void;
+  setQueryText: (queryText: string) => void;
+}> = ({ displayedQuery, isMobile, setResultListVisible, setQueryText }) => {
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  useKeys(["AltLeft", "KeyK"], () => inputRef.current?.focus());
+
+  return (
+    <Box
+      sx={(theme) => ({
+        position: "relative",
+        borderRadius: theme.spacing(0.75),
+        marginRight: theme.spacing(2),
+        border: `1px solid ${theme.palette.gray[30]}`,
+        width: isMobile ? "100%" : undefined,
+        ":focus": {
+          margin: "-1px",
+          border: `2px solid ${theme.palette.blue[70]}`,
+        },
```

Comment:
The styling is being defined here because any styles being applied to `input` do not encompass `endAdornment`. I've used your styles below and added them to `:focus-within` instead to make the styles appear correctly.

Reply:
Why not just use `.Mui-focused` CSS selector instead?